testing/src/main/java/org/oppia/android/testing/VerifyItemDisplayedInRecyclerView.kt

```diff
@@ -0,0 +1,24 @@
+package org.oppia.android.testing
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import org.oppia.android.app.recyclerview.RecyclerViewMatcher.Companion.atPositionOnView
+
+class VerifyItemDisplayedInRecyclerView {
```

Comment:
Check `EditTextInputAction` for reference. 

We can make this class all about `RecyclerView` testing. So, it's not only about testing item at the position. In future, we can have things like item string checking or anything that comes up we can use this class as a central utility for all types of testing for RecyclerView.

Reply:
Ditto here.